src/controllers/favorites.controllers.js

```diff
@@ -2,14 +2,13 @@ import favoriteModel from '../models/favorite';
 
 export const addToFavorite = async (data) => {
   const { userId, houseId } = data;
-  
+
   const user = await userService.getById(userId);
   const house = await houseService.getById(houseId);
```

Comment:
`userService` and `houseService` aren't defined. Please apply the changes from `develop`.